_includes/sidebar.html

```diff
@@ -35,8 +35,8 @@ <h3>Tags</h3>
     {% for resource in page.submission_details.resources %}
       <h3>{{ resource[0] | capitalize }}</h3>
       <ul>
-      {% for url in resource[1] %}
-        <li><a href="{{ url }}">{{ url }}</a></li>
+      {% for child in resource[1] %}
+        <li><a href="{{ child.url }}">{% if child.title %}{{ child.title }}{% else %}{{ child.url }}{% endif %}</a></li>
```

Comment:
You indicated this in your notes on the PR, but just wanted to note that with this change, for the previous contributions to render properly those submissions will need to be edited.